app/javascript/selectors/answersSelector.js

```diff
@@ -1,10 +1,43 @@
 import {
   getAllActiveQuestions,
   getAllActiveQuestionsForCategory,
-  getAllQuestionsByCategoryId
+  getAllActiveQuestionIdsForCategory,
 } from "./questionsSelector";
 import { getCategoriesForBuilding } from "./categoriesSelector";
 
+export function getAnswersForBuilding(buildingId, state) {
+  let answers = state.buildings[buildingId].answers;
+  answers = Object.keys(answers).filter((id) => {
+    let email = answers[id].delegation_email;
```

Comment:
Can you change the filter condition to `!isValidAnswer(answers[id])`? (`isValidAnswer` is a selector method from `answersSelector.js`.) This makes the code cleaner and also makes it easier to fix the logic for identifying empty answers in the future once Ethan's Redux changes are in, since we can just modify the selector method